backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplicationConfiguration.java

```diff
@@ -37,6 +37,9 @@
 
 	/** whether to use the accelerometer. default: true **/
 	public boolean useAccelerometer = true;
+	
+	/** whether to use the gyroscope. default: true **/
+	public boolean useGyroscope = true;
```

Comment:
I don't think we need both Accelerometer and Gyroscope on by default. Gyroscope uses considerably more battery than accelerometer, so we probably want it off by default.